apps/aecore/src/aec_siphash.erl

```diff
@@ -0,0 +1,113 @@
+%%%=============================================================================
```

Comment:
This seems quite a self-contained library: have you considered splitting this in a separate aeternity/something repo?

Reply:
Here I don't think a separate repo is merited, this is just an implementation of a special case of SipHash, not the general thing!